QUEST.tsv

```diff
@@ -2272,24 +2272,24 @@ QUEST_20150717_002271	[Usage Limit]{nl}- Due to a recent incident, we sealed the
 QUEST_20150717_002272	[Method of Usage]{nl}1. Use the Schilt Essences to the portal generator.{nl}2. One person could enter when a certain number of Essences are used.{nl}3. When more than one person enter simultaneously, please ask the chief.
 QUEST_20150717_002273	The essence of spark forms here.{nl}You may get burned so be careful.
 QUEST_20150717_002274	Modestas accepted his defeat and promised to pay back later. When Modestas was about to leave with his horse and wagon, the old Alchemist with a grin Modestas
-QUEST_20150717_002275
-QUEST_20150717_002276
+QUEST_20150717_002275 Story of the Flame/Fire
+QUEST_20150717_002276 
 QUEST_20150717_002277
-QUEST_20150729_002281
-QUEST_20150729_002282
-QUEST_20150729_002283
-QUEST_20150729_002284
-QUEST_20150729_002285
-QUEST_20150729_002286
-QUEST_20150729_002287
-QUEST_20150729_002288
-QUEST_20150729_002289
-QUEST_20150729_002290
-QUEST_20150729_002291
-QUEST_20150729_002292
-QUEST_20150729_002293
-QUEST_20150729_002294
-QUEST_20150729_002295
+QUEST_20150729_002281 Hm.. Have the party members fully gathered yet?
+QUEST_20150729_002282 Welcome to the Mercenary Quest Office. {nl}But.. There aren't appropriate requests yet.{nl} 
+QUEST_20150729_002283 When you return with reliable comrades, when I will give you an appropriate quest.{nl}If there is a request.{nl} 
+QUEST_20150729_002284 There is a good request that just came in.{nl}Do you want to take it?
+QUEST_20150729_002285 Anyway, that is very strange.{nl}If it was Goddess Laima's will, not anyone could undo her seal.
```

Comment:
Original text implied undoing her seal is a very difficult task but not impossible. 
I still don't like how my translation flows, I will come back to it later